sim/mage/ignite.go

```diff
@@ -45,7 +45,7 @@ func (mage *Mage) applyIgnite() {
 		ActionID:    core.ActionID{SpellID: 12654},
 		SpellSchool: core.SpellSchoolFire,
 		DefenseType: core.DefenseTypeMagic,
-		ProcMask:    core.ProcMaskProc,
+		ProcMask:    core.ProcMaskSpellProc,
```

Comment:
I think this one would be `ProcMaskSpellDamageProc` right?

Reply:
I don't think so, because this is the application of the debuff,  the damage portion is taken care of in spell.dealDamageInternal
![image](https://github.com/user-attachments/assets/cae09954-0798-42fc-acab-0e712a67ef49)

Reply:
Oh gotcha, ok